Runtime/src/main/java/com/rakuten/tech/mobile/perf/runtime/internal/RuntimeContentProvider.java

```diff
@@ -42,26 +43,27 @@ public boolean onCreate() {
     RequestQueue queue = new RequestQueue(new NoCache(), new BasicNetwork(new HurlStack()));
     queue.start();
 
-    BatteryInfoStore batteryInfoStore = new BatteryInfoStore(context);
-
+    String relayAppId = Util.getRelayAppId(context);
     String subscriptionKey = Util.getSubscriptionKey(context);
-
     String configUrlPrefix = Util
         .getMeta(context, "com.rakuten.tech.mobile.perf.ConfigurationUrlPrefix");
-    String relayAppId = Util.getRelayAppId(context);
-    ConfigStore configStore = new ConfigStore(context, queue, relayAppId, subscriptionKey,
-        configUrlPrefix);
+
+    BatteryInfoStore batteryInfoStore = new BatteryInfoStore(context);
+    String locationUrlPrefix = Util.getMeta(context,
+        "com.rakuten.tech.mobile.perf.LocationUrlPrefix");
+    LocationStore locationStore = new LocationStore(context, queue, subscriptionKey,
+        locationUrlPrefix);
```

Comment:
pulling the `LocationStore` out of the `if (config != null)` check will cause a bug:
if the we have no config or we decide to not track (lose coin toss vs `enablePercentage`) this will still request location from the location API -> that should not happen.

we def need to change that back

- [ ] only request location if tracking is enabled